frontend/src/scenes/notebooks/Nodes/utils.tsx

```diff
@@ -129,8 +129,8 @@ export function useSyncedAttributes<T extends CustomNotebookNodeAttributes>(
                 }),
                 {}
             )
-
-            props.updateAttributes(stringifiedAttrs)
+            // NOTE: queueMicrotask protects us from TipTap's flushSync calls, ensuring we never modify the state whilst the flush is happening
+            queueMicrotask(() => props.updateAttributes(stringifiedAttrs))
```

Comment:
do we support IE 11?

I think `queueMicrotask` isn't available there

https://caniuse.com/?search=queueMicrotask

Reply:
I would be amazed if we fully support IE... And this isn't our first use of that.

Reply:
I'll add the defaults check to our eslint. Then I can ignore this forever